### DISCUSSION TITLE
Weired Error after saving a CollectionType

### DISCUSSION THREAD

**Discussion by sanojsilva** [strapi/strapi#7466] [Legacy Topics]
I get an error message after saving a new collection type. I have already deleted the cache folder and the build folder and rebuilt the project but no luck so far. Currently, I'm creating the collection types manually by creating new files in the api folder.

![image](https://user-images.githubusercontent.com/31297777/90212973-482e6880-de12-11ea-8126-39f1b58ebc28.png)

**Comment by lauriejim** [comment]
Hello can you please give us more information about the collection type attributes.

**Reply by sanojsilva** [reply]
Yes the attributes are

- type
- deliveryCharge
- serviceCharge

Btw this error happens when I try to create a new collection type or update an existing collection type

**Reply by derrickmehaffy** [reply]
I wasn't able to reproduce the error on v3.1.4, possible there is left over artifacts in the database causing an issue? 

@sanojsilva can you please test on a fresh 3.1.4 project and see if you can reproduce the issue?

**Reply by sanojsilva** [reply]
Yeah, I'm also trying to reproduce this error in other projects but they work fine. Is there a way to get a stack trace or something so that I can find where that error comes from and find the problem

**Reply by derrickmehaffy** [reply]
Can you provide the content types or even better if you have a link to the problem project in Github or something we can clone locally to test.

**Reply by sanojsilva** [reply]
Hey I have created a new strapi project and put my api folder and extensions in there and everything seems to work fine. Collection Types creation from the admin panel has no issue. 
So What could be the problem? Something related to my database? I'm using postgres btw.

**Reply by derrickmehaffy** [reply]
Honestly not, without having any information to reproduce locally there isn't much we can do to help here.
